pkgs/by-name/ku/kubefetch/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+buildGoModule rec {
+  pname = "kubefetch";
+  version = "0.7.2";
+
+  src = fetchFromGitHub {
+    owner = "jkulzer";
+    repo = "kubefetch";
+    rev = "${version}";
+    hash = "sha256-ksnqlfm++Np5I7ZUXvVPgm3T2hHM6x9sAWdmK0Whn5k=";
+  };
+
+  vendorHash = "sha256-qsncOsCxepySJI+rJnzbIGxSWlxMzqShtzcEoJD2UPw=";
+
+  meta = {
+    description = "Neofetch for Kubernetes Clusters";
+    longDescription = "A neofetch-like tool to show info about your Kubernetes Cluster.";
```

```suggestion
    description = "A neofetch-like tool to show info about your Kubernetes Cluster";
```

The `longDescription` can be `description`, and original `description` can be removed.